persistence/upgrades/ravendb-gateway-1to2.md

```diff
@@ -0,0 +1,30 @@
+---
+title: NServiceBus Gateway RavenDB Persistence Upgrade from 1 to 2
+summary: Migration instructions on how to upgrade NServiceBus.Gateway.RavenDB 1 to 2
+component: GatewayRavenDB
+related:
+- nservicebus/gateway
+- nservicebus/gateway/ravendb
+reviewed: 2021-11-25
+isUpgradeGuide: true
+upgradeGuideCoreVersions:
+ - 7
+---
+
+## Cluster-wide transactions support
+
+Starting with NServiceBus.Gateway.RavenDB version 2, the gateway persistence supports RavenDB cluster-wide transactions. RavenDB 5.2 or greater is required for both the RavenDB client and server. Cluster-wide transactions support is disabled by default. To enable it, configure the persister like follows:
+
+```
```

Comment:
This should probably be a snippet no?

Reply:
We don't usually do snippets in upgrade guides, but in this case, it's an easy one.

Reply:
That is new to me. Maybe I've only seen old examples like https://github.com/Particular/docs.particular.net/blob/master/Snippets/ASB/ASB_9/Upgrade7to8.cs. Then don't bother Mauro.